### DISCUSSION TITLE
User Dashboard Q: How can I configure the app links open in a new tab?

### DISCUSSION THREAD

**Discussion by Lockszmith-GH** [goauthentik/authentik#8652] [Q&A] [answered]
The apps in the user dashboard are very useful, is there a way to configure them so that they will open a new tab?
(`<a href=... target="_blank"....`)

**Comment by BeryJu** [comment, marked as answer]
In the application settings under `UI settings` there's an option for `Open in new tab`

**Reply by Lockszmith-GH** [reply]
right in front of my face, but completely missed it - thanks.
